js/about/preferences.js

```diff
@@ -759,13 +766,42 @@ class AboutPreferences extends React.Component {
   }
 
   updateTabFromAnchor () {
-    this.setState({
+    const newState = {
       preferenceTab: this.tabFromCurrentHash
-    })
+    }
+    // first attempt at solving https://github.com/brave/browser-laptop/issues/8966
+    // only handles one param and sets it to true
+    const params = this.hashParams
+    if (params && typeof this.state[params] === 'boolean') {
+      newState[params] = true
+    }
+    this.setState(newState)
   }
 
+  /**
+   * Parses a query string like:
+   * about:preferences#payments?ledgerBackupOverlayVisible
+   * and returns the part:
+   * `about:preferences#payments`
```

Comment:
This comment doesn't seem right. Previously this would return 'payments' not 'about:preferences#payments'.

Reply:
Good catch; fixed 😄